spec/rspec/rails/example/controller_example_group_spec.rb

```diff
@@ -84,7 +83,6 @@ def my_helper
 
       it "calls NamedRouteCollection#route_defined? when it checks that given route is defined or not" do
         expect(routes.named_routes).to receive(:route_defined?).and_return(true)
-        expect(routes.named_routes).not_to receive(:helpers)
```

Comment:
Frankly, I'm not sure what it has to do with helpers.

Reply:
Check the original history with blame, I imagine this is a regression check of some kind

Reply:
This needs some deeper research on how it works with 4.2 and later.
I'll comment this out instead and add a todo, there might be a potential to get rid of some legacy in `controller_example_group.rb`.

Reply:
I'd not comment it out, but feel free to leave a todo to investigate.